src/components/PostDetails.tsx

```diff
@@ -1,118 +1,97 @@
-import React from 'react';
-import { Loader } from './Loader';
+import React, {
+  Dispatch, SetStateAction, useEffect, useState,
+} from 'react';
 import { NewCommentForm } from './NewCommentForm';
+import { getComments, removeComment } from '../utils/fetchClient';
+import { Comment } from '../types/Comment';
+import { Loader } from './Loader';
+import { CommentList } from './CommentList';
```

Comment:
Fix this error. And I can't see this file in commit actually 
<img width="517" alt="image" src="https://user-images.githubusercontent.com/99403198/230338595-df402c31-c236-41ac-9160-6f65b5a665d7.png">